frontend/src/components/ExperimentCollection/ExperimentCollectionDashboard/ExperimentCollectionDashboard.tsx

```diff
@@ -12,16 +12,19 @@ interface ExperimentCollectionDashboardProps {
     participantIdUrl: string | null;
 }
 
-export const ExperimentCollectionDashboard: React.FC<ExperimentCollectionDashboardProps> = ({ experimentCollection, participantIdUrl }) => {
-
+export const ExperimentCollectionDashboard: React.FC<ExperimentCollectionDashboardProps> = ({ experimentCollection, participantIdUrl, totalScore }) => {
+    
     const dashboard = experimentCollection.dashboard;
-    const nextExperimentSlug = experimentCollection.nextExperiment?.slug;
+    const nextExperimentSlug = experimentCollection.nextExperiment?.slug;   
+    
     const headerProps = experimentCollection.theme?.header? {
-        nextExperimentSlug,
+        nextExperimentSlug,        
         collectionSlug: experimentCollection.slug,
-        ... experimentCollection.theme.header
+        ...experimentCollection.theme.header,
```

Comment:
I would pass this as a separate object as more related to the header theme than the specific data & information that needs to be shown. E.g. something like:

```suggestion
        headerTheme: experimentCollection.theme.header,
```